stations.csv

```diff
@@ -24920,7 +24920,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 27046;Comunion;comunion;;;42.7171582;-2.9668538;;t;ES;f;Europe/Madrid;f;;;f;;f;;f;eztgh2;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 27047;Congosto;congosto;;;42.61755;-6.52048;;t;ES;f;Europe/Madrid;f;;;f;;f;;f;eze67w;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 27048;Cortegana;cortegana;;;37.91116;-6.82045;;t;ES;f;Europe/Madrid;f;;;f;;f;;f;eyepsv;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-27049;A Coruña;a-coruna;;;43.37135;-8.396;;t;ES;f;Europe/Madrid;f;;;f;;f;;f;ezdn8z;t;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;La Corogne;;;;;;ア・コルーニャ;라코루냐;;Corunha;;;;拉科魯尼亞
+27049;A Coruña;a-coruna;;;43.37135;-8.396;;t;ES;f;Europe/Madrid;t;;;f;;f;;f;ezdn8z;t;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```

Comment:
@inouire I talked to Mick. It's a fairly old (> 1 year) test. We can probably blindly trust Google Translate for simple translation (like [A Coruña-San Cristovo](https://translate.google.com/?source=osdd#auto/ja/A%20Coru%C3%B1a-San%20Cristovo)), and reuse the parent station's info: fields for more subtle cases.